src/cli/utils/sharepoint.go

```diff
@@ -167,6 +166,7 @@ func FilterSharePointRestoreInfoSelectors(
 	sel *selectors.SharePointRestore,
 	opts SharePointOpts,
 ) {
+	AddSharePointFilter(sel, opts.Library, sel.Library)
```

Comment:
I assume this doesn't do anything yet and that's a follow-up PR?

Reply:
Hmm - the unit test and test output you pasted indicates otherwise. Where does this get used?

Reply:
Ignore this conversation. I looked at the implementation of `AddSharePointFilter`

Reply:
nope, that's the whole logic.